### PR TITLE
chore: clean up synth.py

### PR DESCRIPTION
**Pull request by xiaozhenliu-gg5**

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
 **/node_modules
-**/coverage
-test/fixtures
+**/.coverage
 build/
 docs/
 protos/
+system-test/
```
Comment (JustinBeckwith) [inline .eslintignore:6]: I am a little confused.  Are these hand written changes over top of the generated ignorefile?  This should be overwritten by synth:
https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/node_library/.eslintignore

I think we *want* to lint the `system-test` directory.

File: system-test/install.ts [hunk 2]
```diff
@@ -16,36 +16,34 @@
 // ** https://github.com/googleapis/gapic-generator-typescript **
 // ** All changes to this file may be overwritten. **
 
-import {packNTest} from 'pack-n-play';
-import {readFileSync} from 'fs';
-import {describe, it} from 'mocha';
+import { packNTest } from 'pack-n-play';
```
Comment (JustinBeckwith) [inline system-test/install.ts:19]: Yeah this formatting says it isn't applying `prettier` format rules :)

File: .eslintignore (hunk 1 above)
Comment (xiaozhenliu-gg5) [reply .eslintignore:6]: These are all autogeneraed code (I removed the hacks in synth.py about skipping `eslintignore`)
I thought it's not needed any more, so I tried to remove all hacks and see if it works.
I will try upgrade synthtool and run it again, it should be overwritten by synth properly.

**Comment by xiaozhenliu-gg5** [comment]
Some background @JustinBeckwith , I'm iterating all APIs to check the synth.py hacks, see if there are unneeded surgeries in synth script and clean them up. Thanks

File: system-test/install.ts (hunk 2 above)
Comment (xiaozhenliu-gg5) [reply system-test/install.ts:19]: right, double checking. Thanks!

File: .eslintignore (hunk 1 above)
Comment (xiaozhenliu-gg5) [reply .eslintignore]: Reran synthtool and it's changed back by gts. Thanks!

File: system-test/install.ts (hunk 2 above)
Comment (xiaozhenliu-gg5) [reply system-test/install.ts]: This change is back. Thank you!

**Pull request merged**